ibm/data_source_ibm_iam_account_settings.go

```diff
@@ -117,7 +123,6 @@ func dataSourceIbmIamAccountSettingsRead(context context.Context, d *schema.Reso
 	}
 
 	getAccountSettingsOptions := &iamidentityv1.GetAccountSettingsOptions{}
-	//getAccountSettingsOptions.SetIncludeHistory(d.Get("include_history").(bool))
```

Comment:
Why include_history is removed from option struct...we have it has a optioanl argument

Reply:
This was commented out originally, but I have re-added it now and included it in the list of args in the md file.